website/templates/emails/reviews_submission_confirmation.html.mako

```diff
@@ -9,86 +9,99 @@
     %>
     <tr>
         <td>
-            <div style="margin: 40px; background: white;">
-                <p>Hello ${user.fullname},</p>
-                % if is_creator:
-                    <p>
-                        Your ${document_type}
-                        <a href="${reviewable.absolute_url}">${reviewable.title}</a>
-                        has been successfully submitted to ${reviewable.provider.name}.
+            % if document_type == 'preprint':
```

Comment:
`document type` for preprints can be a bunch of things (it's determined by the `preprint_word` field on the provider). checking for `document_type == 'registration'` is safer